mathml-converters/libs

```diff
@@ -0,0 +1 @@
+/home/andreg-p/Projects/latex-grammar/libs
```

Comment:
Why is this here?

Reply:
this looks like a broken symlink. I think it should be removed